content/en/docs/tasks/configure-pod-container/configure-git-repo.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Configure a Pod with a Git Repo
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+This page shows how to configure a Pod with a Git repo using an
+[Init Container](/docs/concepts/workloads/pods/init-containers/) to provision a
+volume before the Pod's primary container runs. While these examples are specific
+to Git, the overall strategy can be extended to other version control systems.
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+* You need to have a Kubernetes cluster, and the kubectl command-line tool must
+be configured to communicate with your cluster. If you do not already have a
+single-node cluster, you can create one by using
+[Minikube](/docs/getting-started-guides/minikube).
+
+* Familiarize yourself with the material in
+[Init Containers](/docs/concepts/workloads/pods/init-containers/).
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Cloning a Git repo
+
+The [emptyDir](/docs/concepts/storage/volumes/#emptydir) volume type can be used
+to share data between multiple containers in a Pod.
+
+First, define a script for cloning a repo to run in the Init Container:
```

Comment:
Should we also mention https://github.com/kubernetes/git-sync as another approach, when continuous syncing is desired?